FiveCalls/FiveCalls/Dashboard.swift

```diff
@@ -12,24 +12,51 @@ struct Dashboard: View {
     @EnvironmentObject var appState: AppState
 
     @State var showLocationSheet = false
+    @State var showRemindersSheet = false
 
     let op = Operator()
     
     var body: some View {
         NavigationView {
             ScrollView {
                 VStack(alignment: .leading, spacing: 10) {
-                    LocationHeader(location: appState.location, fetchingContacts: appState.fetchingContacts)
-                        .padding(.bottom, 10)
-                        .onTapGesture {
-                            showLocationSheet.toggle()
+                    HStack {
+                        if USE_SHEET {
+                            Button(action: {
+                                showRemindersSheet.toggle()
+                            }, label: {
+                                if let image = UIImage(named: "gear")?.withRenderingMode(.alwaysTemplate)  {
+                                    Image(uiImage: image)
+                                }
+                            })
+                            .sheet(isPresented: $showRemindersSheet) {
+                                ScheduleReminders()
+                            }
+                        } else {
+                            NavigationLink(destination: ScheduleReminders(), label: {
+                                if let image = UIImage(named: "gear")?.withRenderingMode(.alwaysTemplate)  {
+                                    Image(uiImage: image)
+                                }
+                            })
                         }
+                        
+                        LocationHeader(location: appState.location, fetchingContacts: appState.fetchingContacts)
+                            .onTapGesture {
+                                showLocationSheet.toggle()
+                            }
                         // TODO(iOS 16): replace with presentationDetents
-                        .adaptiveSheet(isPresented: $showLocationSheet, detents: [.medium()], smallestUndimmedDetentIdentifier: .none) {
-                            LocationSheet(location: appState.location, delegate: (UIApplication.shared.delegate as! AppDelegate))
-                                .padding(.top, 40)
-                            Spacer()
+                            .adaptiveSheet(isPresented: $showLocationSheet, detents: [.medium()], smallestUndimmedDetentIdentifier: .none) {
+                                LocationSheet(location: appState.location, delegate: (UIApplication.shared.delegate as! AppDelegate))
+                                    .padding(.top, 40)
+                                Spacer()
+                            }
+                        if let image = UIImage(named: "5calls-stars") {
```

Comment:
In non-SwiftUI version, this is currently a button that does nothing.

Reply:
I don't really love the navigation experience for both of these header buttons in the current app, but I *really* don't like the footer buttons, so we can put the about screen behind this one until I have a better idea. Maybe the stats can go there as well for now.

(not really part of this PR but FYI)

Reply:
Would it make sense to convert the settings gear entry point into the typical hamburger menu with settings, about and my impact all included, or does that reduce discoverability too much?